scripts/push.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# (C) 2022 GoodData Corporation
+import os
+from gooddata_sdk import GoodDataSdk
+from pathlib import Path
+from dotenv import load_dotenv
+
+def main():
+    # Get current environment, defaulting to "development"
+    environment = os.getenv("GD_ENV", "development")
+
+    # Load corresponding env variables from file, if exists
+    load_dotenv(f".env.{environment}")
+
+    # Get all the necessary arguments from environment
+    host = os.getenv("GD_HOST")
+    if host is None:
+        raise RuntimeError("GD_HOST environmental variable must be defined")
+
+    token = os.getenv("GD_TOKEN")
+    if token is None:
+        raise RuntimeError("GD_TOKEN environmental variable must be defined")
+
+    credentials_path = os.getenv("GD_CREDENTIALS", f"credentials.{environment}.yaml")
+    if not Path(credentials_path).exists():
+        raise RuntimeError(f"Credentials file does not exist, trying to load from {credentials_path}")
+
+    root_path = Path.cwd()
+
+    sdk = GoodDataSdk.create(host, token)
```

Comment:
We can add check tha the GoodData instance is up. 
`if not sdk.support.is_available`